src/test/java/com/maxmind/minfraud/response/EmailTest.java

```diff
@@ -17,11 +18,13 @@ public void testEmail() throws Exception {
                         .startObject()
                         .put("is_free", false)
                         .put("is_high_risk", true)
+                        .put("first_seen", "2017-01-02")
                         .end()
                         .finish()
         );
 
         assertFalse(email.isFree());
         assertTrue(email.isHighRisk());
+        assertEquals(email.firstSeen(),"2017-01-02");
```

Comment:
Please also add all the new fields to the JSON at https://github.com/maxmind/minfraud-api-java/tree/master/src/test/resources/test-data, which will test serialization and deserialization.